Quadriga/src/main/java/edu/asu/spring/quadriga/rest/SearchNetworkRestController.java

```diff
@@ -0,0 +1,133 @@
+
+import org.apache.velocity.Template;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+import org.apache.velocity.exception.MethodInvocationException;
+import org.apache.velocity.exception.ParseErrorException;
+import org.apache.velocity.exception.ResourceNotFoundException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import edu.asu.spring.quadriga.domain.factories.IRestVelocityFactory;
+import edu.asu.spring.quadriga.domain.workbench.IProject;
+import edu.asu.spring.quadriga.exceptions.QuadrigaAccessException;
+import edu.asu.spring.quadriga.exceptions.QuadrigaStorageException;
+import edu.asu.spring.quadriga.exceptions.RestException;
+import edu.asu.spring.quadriga.service.IRestMessage;
+import edu.asu.spring.quadriga.service.network.INetworkTransformationManager;
+import edu.asu.spring.quadriga.service.network.domain.ITransformedNetwork;
+import edu.asu.spring.quadriga.service.workbench.IRetrieveProjectManager;
+import edu.asu.spring.quadriga.transform.Node;
+
+/**
+ * Controller for concept related rest api's exposed to other clients Client
+ * 
+ * @author Suraj Nilapwar
+ * 
+ */
+@Controller
+public class SearchNetworkRestController {
+
+    private static final Logger logger = LoggerFactory.getLogger(SearchNetworkRestController.class);
+
+    @Autowired
+    private IRestMessage errorMessageRest;
+
+    @Autowired
+    private IRestVelocityFactory restVelocityFactory;
+    
+    @Autowired
+    private IRetrieveProjectManager projectManager;
+    
+    @Autowired
+    private INetworkTransformationManager transformationManager;    
+    
+    /**
+     * Rest interface to search concept in list of projects
+     * http://<<URL>:<PORT>>/quadriga/rest/networks/search
+     * http://localhost:8080/quadriga/rest/networks/search?conceptId=http://www.digitalhps.org/concepts/CON516ec8c2-20bc-4f35-ae1e-ad908db9d662&projectIds=PROJfFnfnn,PROJFgGPpk
+     * 
+     * @param conceptId: ConceptId for searching node
+     * @param projectIds : List of comma separated projectId
+     * @param response
+     * @param accept
+     * @param principal
+     * @return status
+     * @throws RestException
+     */
+    
+    @RequestMapping(value = "rest/network/search", method = RequestMethod.GET, produces = "application/xml")
+    public ResponseEntity<String> getSearchTransformedNetwork(@RequestParam("conceptId") String conceptId, @RequestParam("projectIds") List<String> projectIds,

```

Comment:
ok, one last thing, sorry, I didn't think of this earlier. A user needs to be authenticated to use this method and you have to make sure the user is owner or collaborator on the provided projects.